### DISCUSSION TITLE
How to specify the perspective sensor directly using an intrinsic matrix?

### DISCUSSION THREAD

**Discussion by 07hyx06** [mitsuba-renderer/mitsuba3#708] [Q&A] [answered]
Hi, I find that the document suggests specifying the perspective camera via fov or some other parameters ([Here](https://mitsuba.readthedocs.io/en/latest/src/generated/plugins_sensors.html#perspective-pinhole-camera-perspective)). I wonder is there exists some API to directly set the perspective camera from the intrinsic matrix, i.e. [[fx, 0, cx], [0, fy, cy], [0, 0, 1]]? 

Thanks.

**Comment by njroussel** [comment, marked as answer]
Hi @07hyx06 
No, we do not provide such a utility function.
